src/admin/components/SearchPage.js

```diff
@@ -30,9 +30,9 @@ export default class AdminSearchPage extends React.Component {
     } = this.props;
     const pathname = '/search/';
     const paginator = hasSearchParams && count > 0 ?
-      <Paginate count={count} currentPage={page}
+      (<Paginate count={count} currentPage={page}
```

Comment:
It does seem weird to wrap what boils down to a function call in parentheses. I think I had ran into a ternary where breaking right before the ? and : made it pass lint. I kind of like that form.